sdk/tables/aztable/connection_string.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztable
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+// NewClientFromConnectionString creates a new Client struct from a connection string. The connection
+// string must contain either an account name and account key or an account name and a shared access signature.
+func NewClientFromConnectionString(connectionString string, tableName string, options *ClientOptions) (*Client, error) {
```

Comment:
I'd remove this function

Reply:
Are we expecting a user to parse the connection string themself?

Reply:
No, keep the other NewServiceClientFromConnectionString function; just kill this one.
If they have a cxn str then they have to create the service client first and then they can call the method to get back a Client. We're not enabling anything with NewClientFromCxnStr that can't be done another way.